flyio/caveats.md

```diff
@@ -0,0 +1,405 @@
+
+# Macaroons and Caveats at Fly
+
+This document covers the general mechanisms of Macaroons as they are used at
+Fly to do access control. It does not cover the cryptographic constructs.
+
+Macaroons are tokens. They are define in https://github.com/superfly/macaroon.
+At fly we pass them to services in `Authorization: Bearer <token>` HTTP headers.
```

Comment:
```suggestion
At fly we pass them to services in `Authorization: FlyV1 <token>` HTTP headers.
```

Reply:
changing, but I did see flyctl passing the token in a "Bearer"
FLY_API_BASE_URL=http://localhost:8080/ flyctl app list -> nc -l 8080 I see:
Authorization: Bearer fo1_XXXXXX

Reply:
OAuth tokens (`fo1_...`) are supposed to go in `Bearer` and macaroons are supposed to go in `FlyV1`. We're super flexible about this though.